wp-content/themes/missilethreat/assets/_scss/abstracts/_placeholders.scss

```diff
@@ -4,3 +4,149 @@
 /*==============================
 =          Placeholders        =
 ==============================*/
+%font-ui-text-sm {
+  font-size: 12px;
+  line-height: 1.14;
+}
+
+%font-ui-text-md {
+  font-size: 14px;
+  line-height: 1.14;
+}
+
+%font-ui-text-lg {
+  @include font-clamp(16px, 17px);
+  line-height: 1.14;
+}
+
+%font-ui-text-md-semibold-mobile-nav {
+  font-size: 16px;
+  line-height: 1;
+}
+
+%font-ui-page-title-sm {
+  @include font-clamp(28px, 33px);
+  line-height: 1.14;
+}
+
+%font-ui-page-title-md {
+  @include font-clamp(36px, 42px);
+  line-height: 1.14;
+}
+
+%font-ui-page-title-lg {
+  @include font-clamp(42px, 54px);
+  line-height: 1.14;
+}
+
+%font-ui-heading-sm {
+  font-size: 17px;
+  line-height: 1.14;
+}
+
+%font-ui-heading-md {
+  @include font-clamp(20px, 24px);
+  line-height: 1.14;
+}
+
+%font-ui-heading-md-bold {
```

Comment:
@urchykli Can you take a closer look at this one? My gut feeling is that the `font-ui-heading-md-bold` is a mistake, and the size should actually match that of the `font-ui-heading-md` (which means we can remove the `-bold` one).